src/components/common/Drops/styles.js

```diff
@@ -115,7 +132,9 @@ export default theme => ({
       marginTop: 10
     },
     hideOnSmall: {
-      display: 'inline-block'
+      display: 'inline-block',
```

Comment:
The hideOnSmall should only be concerned about what the name says. If you want to set the dimensions, use other class.